README.md

```diff
@@ -1,16 +1,162 @@
-# Title
+# Return Values
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Identify implicit return values in Ruby syntax
+- Recognize the explicit `return` keyword
 
 ## Introduction
 
-## SWBAT 1
+Displaying text in the console is not the only way available to get information
```

Comment:
this first sentence feels murky to me. return values aren't really like output. i'd remove.